src/__tests__/reservations/reservation.test.js

```diff
@@ -0,0 +1,13 @@
+import { BrowserRouter } from "react-router-dom";
+import AddReservation from "../../components/reservations/reservation";
+import renderer from 'react-test-renderer';
+
+test('Reservation form match snapshot', () => {
+    const component = renderer.create(
+        <BrowserRouter>
+            <AddReservation />
+        </BrowserRouter>
+    );
+    let tree = component.toJSON();
+    expect(tree).toMatchSnapshot();
+});
```

Comment:
- [ ] There are still some issues with the frontend tests:

![image](https://user-images.githubusercontent.com/57421823/213449385-32b72dbc-7bbb-4564-854a-a28c3be319da.png)

![image](https://user-images.githubusercontent.com/57421823/213449444-59cdaeee-b373-40d8-bbca-456ed5933389.png)

![image](https://user-images.githubusercontent.com/57421823/213449486-c5413256-9fa3-4c5a-a74b-71138fb80f31.png)

There is a test suite that is not running and a failed test.  

Kindly, follow the error messages and ensure that all your tests are passing.

You've come all this way, to the top of the Microverse program.  I'm pretty sure you are able to power through this small issue! 💪